interceptors/logging/payload.go

```diff
@@ -31,18 +24,30 @@ func (c *serverPayloadReporter) PostMsgSend(req interface{}, err error, duration
 	if err != nil {
 		return
 	}
+
+	p, ok := req.(proto.Message)
+	if !ok {
+		// TODO(bwplotka): Panic / log err?
```

Comment:
For now, we should log the error made, else this would make the life of users difficult, wondering why the logging middleware isn't working.